tests/logout.test.js

```diff
@@ -0,0 +1,16 @@
+import { logout } from "../src/js/api/auth/logout.js";
+import "/Users/hiboharun/Documents/GitHub/social-media-client/.setupTest.js"; // Absolute path
```

Comment:
Why was this necessary? It will not work on anyone else's computer other than yours unfortunately.